### DISCUSSION TITLE
Date Validation Error

### DISCUSSION THREAD

**Discussion by supermx1** [directus/directus#7958] [Feature Requests]
So, I am starting to suspect this might be a bug or missing feature... 
For some weeks now I have been trying to validate a `date` field in the Roles & Permissions section by using filter rules and no matter what I try it either throws this error

![Screen Shot 2021-09-09 at 10 57 20 PM](https://user-images.githubusercontent.com/9326922/132768040-2f21b709-bc7d-47e2-a1cb-84c70875caa6.jpg)

Here is my validation filter

```json
{
    "_or": [
        {
            "date_of_birth": {
                "_gte": "2004-09-09"
            }
        },
        {
            "date_of_birth": {
                "_lte": "2021-09-09"
            }
        }
    ]
}
```

Now I have tried every combination including using dynamic variables like $NOW and $NOW(-17 years). It either throws the above error and the console logs this:

![Screen Shot 2021-09-09 at 11 00 19 PM](https://user-images.githubusercontent.com/9326922/132768328-91cd1e35-361b-479b-895c-5aa64d92d147.jpg)

or it throws 

![Screen Shot 2021-09-09 at 11 06 23 PM](https://user-images.githubusercontent.com/9326922/132768838-626802af-24a8-417e-95d4-c411f467d4d1.jpg)

I am on rc92, MariaDB, MacOS,Chrome

**Comment by supermx1** [comment]
I have tried validation filters like the following just to see if one sticks:

```json
{
    "date_of_birth": {
        "_between": "2004-09-09,2021-09-09"
    }
}
```

```json
{
    "_or": [
        {
            "date_of_birth": {
                "_gte": "2004-09-09"
            }
        },
        {
            "date_of_birth": {
                "_lte": "2021-09-09"
            }
        }
    ]
}
```

```json
{
    "date_of_birth": {
        "_between": [
            "2004-09-09",
            "2021-09-09"
        ]
    }
}
```

```json
{
    "date_of_birth": {
        "_lte": "$NOW"
    }
}
```

Now all I am trying to do is validate a date field to make sure the field can only accept a date of birth between the current date and/or not older than 17years of age... That is simply all.... For some reason, nothing works.

FYI: Using $NOW doesn't work too

**Comment by rijkvanzanten** [comment]
Did you run the db migrations after updating to rc.92?

**Reply by supermx1** [reply]
Yes. I did.

**Reply by rijkvanzanten** [reply]
> Yes. I did. 

Can you share a database dump? That limit field as removed completely in 92 (in https://github.com/directus/directus/pull/7711) 🤔

**Reply by supermx1** [reply]
https://drive.google.com/file/d/122xrb4af7V8rUd0iM4vm8sVHaZNmMyxl/view?usp=sharing

**Reply by rijkvanzanten** [reply]
yeah that seems to be a bug in validation specifically, moved it to https://github.com/directus/directus/issues/8214 👍🏻
